src/MainControlLoop.cpp

```diff
@@ -23,24 +23,21 @@ MainControlLoop::MainControlLoop()
       burnwire_control_task(constants::timecontrol::burnwire_control_task_offset),
       camera_control_task(constants::timecontrol::camera_control_task_offset),
       rockblock_control_task(constants::timecontrol::rockblock_control_task_offset),
-      eeprom_control_task(),
+      eeprom_control_task(constants::timecontrol::eeprom_control_task_offset),
       mission_manager(constants::timecontrol::mission_manager_offset)
-
 {
     delay(1000);
 }
 
 void MainControlLoop::execute()
 {
-    delay(200);
+    delay(200); // To prolong the speed of the main control loop to ensure correct RockBlock reads. Can reduce in the future.
```

Comment:
I would open a follow up ticket or subticket to investigate best control loop increase amount

Reply:
I will open up a ticket after this is merged